pkgs/misc/urbit/ca-bundle.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenvNoCC, xxd, cacert }:
+
+stdenvNoCC.mkDerivation {
+  name = "ca-bundle";
+
+  nativeBuildInputs = [ cacert xxd ];
+
+  phases = [ "installPhase" ];
+
+  installPhase = ''
+    set -euo pipefail
+
+    if ! [ -f "$SSL_CERT_FILE" ]; then
+      header "$SSL_CERT_FILE doesn't exist"
+      exit 1
+    fi
+
+    mkdir include
+
+    cat $SSL_CERT_FILE > include/ca-bundle.crt
+    xxd -i include/ca-bundle.crt > ca-bundle.h
```

Comment:
```suggestion
    cat $SSL_CERT_FILE > ca-bundle.crt
    xxd -i ca-bundle.crt > ca-bundle.h
```

Reply:
`include/` is needed because output in C include file style is named after input file.